include/library/spdm_common_lib.h

```diff
@@ -218,6 +220,12 @@ typedef enum {
 #define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_LITTLE_ONLY     (1)
 #define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_BIG_OR_LITTLE   (2)
 
+typedef enum {
+    LIBSPDM_DATA_SESSION_SEQ_NUM_ENC_LITTLE_DEC_LITTLE,
+    LIBSPDM_DATA_SESSION_SEQ_NUM_ENC_BIG_DEC_BIG,
+    LIBSPDM_DATA_SESSION_SEQ_NUM_ENC_LITTLE_DEC_BOTH,
+    LIBSPDM_DATA_SESSION_SEQ_NUM_ENC_BIG_DEC_BOTH
+} libspdm_sequence_number_endian_t;
```

Comment:
Can we use consistent way to handle session endianness and signature endianness?

I am ok with either way - enum or uint8_t.

Reply:
Sure, I can change it to `uint8_t` and use macros.